Weather/weather-alerts.1min.ts

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env -S -P/${HOME}/.deno/bin:/opt/homebrew/bin deno run --allow-net 
```

Comment:
@theogainey 

Would it make sense to add `/usr/local/bin` (the non-M1 silicon Homebrew path) here?

Reply:
Yes it would so I added it